contracts/tokens/Xcert.sol

```diff
@@ -94,6 +123,23 @@ contract Xcert is NFTokenMetadata {
     return idToProof[_tokenId][idToProof[_tokenId].length.sub(1)];
   }
 
+  /*
+   * @dev Gets metadata for NFT token.
+   * @param _tokenId Id of the NFT.
+   * @param _index Metadata position.
+   */
+  function tokenMetadata(
```

Comment:
`getTokenMetadata` is better

Reply:
True, but this is to be aligned with standard's naming.